code/datums/brain_damage/mrat.dm

```diff
@@ -122,6 +122,8 @@
 	leave.Grant(src)
 
 	grant_all_languages()
+	grant_language(/datum/language/metalanguage)
```

Comment:
this should be set to the default language too. Either way it's only visible to the person hearing it. No need to grant all languages in this case either.

Imaginary friends should act the same imo

Reply:
imaginary friends are a kind of their another personality which means they should inherent the language of their 'original'.
mentor rat is fine with it, so I will update.